webapp/src/client.js

```diff
@@ -15,29 +15,36 @@ export default class Client {
         return this.doPost(`${this.url}/settings`, meeting);
     }
 
-    doGet = async (url, body, headers = {}) => {
-        headers['X-Requested-With'] = 'XMLHttpRequest';
-        headers['X-Timezone-Offset'] = new Date().getTimezoneOffset();
+    doGet = async (url, headers = {}) => {
+        return this.doFetch(url, { headers });
```

Comment:
Not a required change, just something that matches our usual conventions.

```suggestion
        return this.doFetch(url, {headers});
```